src/filters/i18n-validation-exception.filter.ts

```diff
@@ -17,31 +17,40 @@ import {
   I18nValidationException,
 } from '../interfaces/i18n-validation-error.interface';
 import { getI18nContextFromArgumentsHost } from '../utils/util';
+import { Request, Response } from "express";
+import { GqlExceptionFilter } from '@nestjs/graphql';
 
 type I18nValidationExceptionFilterOptions = Either<
   I18nValidationExceptionFilterDetailedErrorsOption,
   I18nValidationExceptionFilterErrorFormatterOption
 >;
 
 @Catch(I18nValidationException)
-export class I18nValidationExceptionFilter implements ExceptionFilter {
+export class I18nValidationExceptionFilter implements ExceptionFilter,GqlExceptionFilter {
   constructor(
     private readonly options: I18nValidationExceptionFilterOptions = {
       detailedErrors: true,
     },
   ) {}
   catch(exception: I18nValidationException, host: ArgumentsHost) {
     const i18n = getI18nContextFromArgumentsHost(host);
-    const response = host.switchToHttp().getResponse<any>();
+    const response = host.switchToHttp().getResponse<Response>();
```

Comment:
I think it would be better to implement it like how we did in the resolvers.

Example:
```typescript
switch (context.getType() as string) {
      case 'http':
        req = context.switchToHttp().getRequest();
        service = req.i18nService;
        break;
      case 'graphql':
        [, , { req, i18nService: service }] = context.getArgs();
        if (!req) return undefined;
        break;
      default:
        return undefined;
    }
```